frontend/src/routes/(authenticated)/myaccount/+page.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+    // This script handles the account settings page.
+    // For now, it only allows you to modify the users display name
+    // Minimal changes allow it to modify the users email as well (but this should have a re-verification system)
+    import type { PageData } from './$types';
+    import t from '$lib/i18n';
+    import { Button, Form, Input, lexSuperForm } from '$lib/forms';
+    import { Page } from '$lib/layout';
+    import { user } from '$lib/user';
+    import {_changeUserAccountData} from "./+page";
+    import type {ChangeUserAccountDataInput} from '$lib/gql/types';
+
+    // Get users data (reactive)
+    $: users_name = $user?.name; // not to be confused with username
+    $: email = $user?.email;
+    $: userid = $user?.id;
+    let newName: string = users_name || '';
+    let changed = false;
+
+    // This function updates the account information on the server
+    async function updateAccount(
+      email: string | undefined,
+      name: string | null,
+    ) {
+      if (confirm($t('account_settings.confirm_change'))) {
+        // prepare the input
+        const changeUserAccountDataInput: ChangeUserAccountDataInput = {
+            email: email || '',
+            name: name || '',
+            userId: userid || '',
+        };
+        _changeUserAccountData(changeUserAccountDataInput);
+        changed = true;
+      }
+    }
+   export let data: PageData;
+</script>
+
+<Page>
+    <svelte:fragment slot="header">
+        {$t('account_settings.title')}
+    </svelte:fragment>
+    <div class="content-center">
+        <h1 class="card-title justify-center text-3xl mt-10 mb-28">{$t('account_settings.page_h1')}</h1>
+        <Form>
+            <Input
+                id="email"
+                label={$t('account_settings.label_email')}
+                type="email"
+                bind:value={email}
+                autofocus
+                placeholder={email}
+                readonly={true}
+            />
+            <Input
+                id="name"
+                label={$t('account_settings.label_name')}
+                type="text"
+                bind:value={newName}
+                placeholder={users_name}
```

Comment:
How about we just use `{$user?.name}` here and toss the `users_name` field.
The same goes for the email placeholder.